amf-cli/shared/src/test/resources/upanddown/cycle/async20/bindings/gps-message-binding.yaml

```diff
@@ -0,0 +1,28 @@
+asyncapi: 2.5.0
+info:
+  title: Simple API with GooglePubSub Message Binding
+  version: 1.0.0
+servers: {}
+channels:
+  userEventChannel:
+    subscribe:
+      message:
+        $ref: "#/components/messages/UserEvent"
+components:
+  messages:
+    UserEvent:
+      summary: A message for user events.
+      payload:
+        type: object
+        properties:
+          userId:
+            type: string
+            description: Unique identifier for the user.
+          action:
+            type: string
+            description: Description of the user action.
+      bindings:
+        googlepubsub:
+          attributes:
+            exampleAttribute: exampleValue
+          orderingKey: userEventOrder
```

Comment:
Add `schema` key to the test. We should test all the fields

Reply:
Also, add one ChannelBinding testing the same, in the 3 new models

Reply:
added all schemas to test in message and channels